sebs/azure/azure.py

```
@@ -69,26 +144,40 @@ def __init__(
         self.logging_handlers = logger_handlers
         self._config = config
 
-    """
-        Start the Docker container running Azure CLI tools.
-    """
-
     def initialize(
         self,
         config: Dict[str, str] = {},
```

Ruff (0.15.2)

[warning] Unused method argument: `config`

(ARG002)